### PR TITLE
C# operators: update links to the code examples

### PR DESCRIPTION
**Pull request by pkulikov**
Supported by dotnet/samples#956

### CODE REVIEW

**Comment by BillWagner** [comment]
closing and re-opening for a fresh build.

File: docs/csharp/language-reference/operators/true-false-operators.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ A type with the defined `true` operator can be the type of a result of a control
 
 The following example presents the type that defines both `true` and `false` operators. Moreover, it overloads the logical AND operator `&` in such a way that the operator `&&` also can be evaluated for the operands of that type.
 
-[!code-csharp-interactive[true and false operators example](~/samples/snippets/csharp/keywords/TrueFalseOperatorsExample.cs)]
+[!code-csharp-interactive[true and false operators example](~/samples/csharp/language-reference/operators/TrueFalseOperators.cs)]
```
Comment (BillWagner) [inline docs/csharp/language-reference/operators/true-false-operators.md]: This is unrelated to this PR:

The current limitation where docs.microsoft.com scaffolds all interactive samples means that this one does not work.  Can you change it from `code-csharp-interactive` to `code-csharp`? 

We can turn in back on when the features to support full programs are delivered.

```suggestion
[!code-csharp[true and false operators example](~/samples/csharp/language-reference/operators/TrueFalseOperators.cs)]
```

**Review by BillWagner — CHANGES_REQUESTED** [review]
Overall, this is great @pkulikov 

I do have one comment, unrelated to this PR, but a bug due to current platform limitations. Once that's addressed, I'll :shipit:

**Review by BillWagner — APPROVED** [review]

**Comment by pkulikov** [comment]
@BillWagner thanks for the notice; I have applied your suggestion.
